index.js

```diff
@@ -634,7 +657,7 @@ function httpViewCleanup(db) {
   });
 }
 
-var localViewCleanup = utils.sequentialize(mainQueue, function (db) {
+function localViewCleanup(db) {
```

Comment:
The tests should be catching this if I use it incorrectly, since I've got some parallel map/reduce operations going on. But I could probably stand to add some more (e.g. making many views simultaneously) just to be safe.